src/Badge/index.d.ts

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import { BadgeProps } from 'react-bootstrap/Badge';
+
+declare const Badge: React.ForwardRefExoticComponent<BadgeProps>;
```

Comment:
After the test, I see in the Badge component a problem with `children` props.
Maybe, we will create a BadgeProps interface, then extend him types React Bootstrap?